packages/rich-text-structure/src/is-collapsed.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Check if the selection of a rich text record is collapsed.
+ *
+ * @param {Object} record The rich text record to check.
+ *
+ * @return {boolean} True if the selection is collapsed, false if not.
+ */
```

Comment:
what does "collapsed" mean? when could we encounter it?

is this mainly checking if a selection range is a zero-width caret vs. a range?

Reply:
Yes, exactly. It's just like `window.getSelection().isCollapsed()`.